src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 6.2
 ---
 
+ * Add support for UUID v7 as default uuid version
```

Comment:
Not needed for me.

Reply:
removed again. i thought it was a requirement as it was stated in the pull-request template